Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitTestHelper.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+/// Provides helper functions to test the SEPA Direct Debit flow in sandbox.
+@objcMembers public class BTSEPADirectDebitTestHelper: NSObject {
+
+    /// Generates a valid 27-digit IBAN (International Bank Account Number) for testing SEPA Direct Debit flows.
+    /// - Returns: a valid IBAN
+    public static func generateValidIBAN() -> String {
```

Comment:
Can a merchant use this for prod testing? If not, we can specify only sandbox.

Reply:
Good call, this is sandbox only - will add sandbox to the method signature!